backend/utils.py

```diff
@@ -96,6 +143,7 @@ def call_github_action(
     return response
 
 
+# TODO: I don't think we're using this. Delete?
```

Comment:
I left it here as a reminder, but yeah that's fine. I have an alternative method for alerting us anyway, which I haven't implemented, but it involves purposefully failing actions.